index.html

```diff
@@ -66,16 +66,16 @@
    <center><p>Hi, I am Yoshitha Rathnayake. This is the Calculator App that I made using Python Programming Language. You can download it by clicking the below link. It's saying there are viruses, but don't worry they are lies.</p></center>
     
 
-    <img src="assets/img/CALCULATOR.png" height="100px" width="100px">
     
 
     <center>
         <img src="assets/img/CALCULATOR.png">
-    </center>
+    
+        <br>
+        <br>
         
-
    <a href="Application/CALCULATOR.exe" class="btn btn-success btn-sm"><i class="fa-solid fa-download"></i> Download</a>
-  
+</center>
```

Comment:
Please add the center to the image only bcs if you put it to the download btn a chance of another will missed out @DivyanshVinayak23